lib/Net/Async/Blockchain/BTC.pm

```diff
@@ -0,0 +1,221 @@
+package Net::Async::Blockchain::BTC;
+
+use strict;
+use warnings;
+
+our $VERSION = '0.001';
+
+=head1 NAME
+
+Net::Async::Blockchain::BTC - Bitcoin based subscription.
+
+=head1 SYNOPSIS
+
+    my $loop = IO::Async::Loop->new;
+
+    $loop->add(
+        my $btc_client = Net::Async::Blockchain::BTC->new(
+            subscription_url => "tcp://127.0.0.1:28332",
+            rpc_url => 'http://test:test@127.0.0.1:8332',
+            rpc_timeout => 100,
+        )
+    );
+
+    $btc_client->subscribe("transactions")->each(sub { print shift->{hash} })->get;
```

Comment:
I'm guessing this is an RYU stream?  might pay to add that to the description otherwise it is difficult to know what methods could be called on it.

Reply:
@code4pay I fixed the return in the subscribe method, it should be clear now: [here](https://github.com/binary-com/perl-Net-Async-Blockchain/pull/1/commits/4d44b004eb9befffef4c9cbfdddab9823054ea74#diff-73116024955ac2972658773dc1ecd4f4R112)